src/NuGet.Clients/NuGet.Tools/OutputConsoleLogger.cs

```diff
@@ -130,15 +130,23 @@ public void Log(ILogMessage message)
         private async Task<int> GetMSBuildVerbosityLevelAsync()
         {
             await NuGetUIThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
+            int verbosity = DefaultVerbosityLevel;
```

Comment:
I did this change because every time I run the PM UI in a experimental instance, it always fails with the following message

```shell
System.Runtime.InteropServices.COMException (0x80020009): Exception occurred. (Exception from HRESULT: 0x80020009 (DISP_E_EXCEPTION))
   at EnvDTE.Property.get_Value()
   at NuGetVSExtension.OutputConsoleLogger.<GetMSBuildVerbosityLevelAsync>d__23.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at NuGetVSExtension.OutputConsoleLogger.<<Start>b__24_0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at Microsoft.VisualStudio.Threading.JoinableTask.CompleteOnCurrentThread()
   at Microsoft.VisualStudio.Threading.JoinableTaskFactory.Run(Func`1 asyncMethod)
   at NuGetVSExtension.OutputConsoleLogger.Start()
   at NuGet.PackageManagement.UI.NuGetUI.BeginOperation()
   at NuGet.PackageManagement.UI.UIActionEngine.<>c__DisplayClass12_0.<<PerformActionImplAsync>b__0>d.MoveNext()
Time Elapsed: -10675199.02:48:05.4775808
========== Finished ==========
```

Reply:
I moved that type (OutputConsoleLogger) recently...into N.VS.Common.
I also changed the namespace to NuGet.VisualStudio.Common
We were a little liberal with moving types without forwarding addresses, and changing the namespaces -- since they are nuget.client based types.

did my change cause this problem?

Reply:
I tried to debug NuGet.Client 1 commit before MEF moving changes (f5a668f8f16f071957caba149195f6cebc39e74a) and I was not able to open PM UI. Exception details below:

System.NullReferenceException
  HResult=0x80004003
  Message=Object reference not set to an instance of an object.
  Source=NuGet.Tools
  StackTrace:
   at NuGetVSExtension.NuGetPackage.<>c__DisplayClass85_0.<<BeforeQueryStatusForAddPackageForSolutionDialog>b__0>d.MoveNext() in C:\Users\FEAGUILA\msft\NuGet.Client.Fresh\src\NuGet.Clients\NuGet.Tools\NuGetPackage.cs:line 1000

Reply:
Fixed by external team. See https://dev.azure.com/devdiv/DevDiv/_git/VS/pullrequest/248291?_a=files